javascript/javascript2/week2/number-double.js

```diff
@@ -0,0 +1,14 @@
+let numbers = [1, 2, 3, 4];
+let newNumbers = [];
+let swapArray = [];
+
+numbers.forEach((value, index) => {
```

Comment:
this code it's way too complicated for this exercise :)

Try to think what `map `and `filter `methods do . What this methods return...and think of chaining.

`result = myArray.reduce(...).map(...)`